dali/python/nvidia/dali/plugin/base_iterator.py

```diff
@@ -278,7 +278,7 @@ def check_all_or_none_and_get(input_meta, name, err_msg):
             self._shard_sizes_per_gpu = self._calculate_shard_sizes(np.arange(0, self._shards_num))
 
             # to avoid recalculation of shard sizes when iterator moves across the shards
-            # memorize the initial shard sizes and then use chaning self._shards_id to index it
+            # memorize the initial shard sizes and then use changing self._shards_id to index it
```

Comment:
```suggestion
            # memorize the initial shard sizes and then change self._shards_id to index it
``?

Reply:
I looked into the code and the logic there is that `size._shards_id` is used as an index of `self._shard_sizes_per_gpu` and `size._shards_id` is changing frequently. So the meaning here was to _use self._shards_id, which will be changing frequently_, so `size._shards_id` is _used_ and is _changing_.